tools/contributors-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# These are pip requirements for contributors.py. They were extracted from
```

Comment:
Let's remove this file. Having more files to maintain is more costly than five seconds.

Reply:
Deal!